cmake/modules/AddSwift.cmake

```diff
@@ -536,8 +535,29 @@ function(add_swift_host_library name)
         "LINKER:-current_version,${SWIFT_COMPILER_VERSION}")
     endif()
 
+    # Always link as CXX even if we have swift content unless we only contain
+    # swift content signaled via us being marked "PURE_SWIFT".
+    if (NOT ASHL_PURE_SWIFT)
+      set_target_properties(${name} PROPERTIES LINKER_LANGUAGE CXX)
+    endif()
+
+    # If we found a swift compiler and are going to use swift code in swift host
+    # side tools but link with clang, add the appropriate -L paths so we find
+    # all of the necessary swift libraries.
+    if (CMAKE_Swift_COMPILER)
+      # Add in the SDK directory for the host platform and add an rpath.
+      target_link_libraries(${name} PRIVATE
+        -L${SWIFT_SDK_${SWIFT_HOST_VARIANT_ARCH}_ARCH_${SWIFT_HOST_VARIANT_ARCH}_PATH}/usr/lib/swift)
```

Comment:
I think this is incorrect. The first thing should be a different value. I was just careless. Fixing!